packages/orbit-design-tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kiwicom/orbit-design-tokens",
-  "version": "4.2.1",
+  "version": "5.0.0-beta.1",
```

Comment:
How will lerna handle the versioning on the next release? :thinking:

Reply:
I think I will exclude those packages from Lerna for a while since it's not generating the changelog properly, and better will be to generate those after we will get them out of beta :) It should be soon, after I will add docs, we can release the first major